TrackPropagation/RungeKutta/test/rkTest_cfg.py

```diff
@@ -25,10 +25,12 @@
 
 #process.load("Configuration.StandardSequences.MagneticField_35T_cff")
 
-process.load("Configuration.StandardSequences.MagneticField_38T_cff")
+#process.load("Configuration.StandardSequences.MagneticField_38T_cff")
 
 #process.load("Configuration.StandardSequences.MagneticField_40T_cff")
 
+process.load("MagneticField.Engine.volumeBasedMagneticField_090322_2pi_scaled_cfi")
```

Comment:
It's no longer relevant here, but just for the record: the idea is that users should not rely on a specific implementation of the map, otherwise when we need to move to a new one we have to track and handle all places it is used. That happened at the start of Run II when we had to search and fix all configurations that were hardcoding a specific map instead of taking the one from the standard sequences. Think of it as hardcoding the db path of a specific calibration which may become obsolete and even removed at some time.
This said, I am not so strict about this, as I mentioned in the first place and as Vincenzo noted; especially for technical tests where the actual field values are not relevant.
BTW MagneticField/Engine/test/queryField.py is mostly used for testing different implementations, and that's why it includes paths to configuration for these specific  implementations.